packages/e2e-test-utils/src/visitSettings.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import visitDashboard from './visitDashboard';
+
+/**
+ * Creates a new story.
+ */
+async function visitSettings() {
```

Comment:
Nitpic, can we use this util here
https://github.com/google/web-stories-wp/blob/9f59605642ac3bcd01fd71e8de897700098d8240/packages/e2e-tests/src/specs/dashboard/integrations/sitekit.js#L37-L49